packages/routes/src/RouteTest.tsx

```diff
@@ -5,7 +5,7 @@ import * as httpMocks from 'node-mocks-http'
 import { Route } from './Route'
 import { RouteContextMock } from './RouteContextMock'
 
-export type HttpMethod = 'GET' | 'POST'
+export type HttpMethod = 'GET' | 'POST' | 'get' | 'post'
```

Comment:
what about 'Post' or 'posT' :)

... just kidding

Reply:
- would need to change every occurence in unicorn / octopus
- express works with both ways
- dont want to make the decision what to use